pypika/terms.py

```diff
@@ -222,7 +222,8 @@ def _get_value_sql(self):
         if isinstance(self.value, date):
             return "'%s'" % self.value.isoformat()
         if isinstance(self.value, basestring):
-            return "'%s'" % self.value
+            value = self.value.replace("'", "''")
+            return "'%s'" % value
```

Comment:
Ah, this should code eventually escape all SQL characters, but haven't managed to find time to implement that yet.